While pbr is used during the runtime, it's not listed as one of the dependenicies.

This was reported upstream:
https://github.com/PyCQA/bandit/issues/735

--- bandit-1.7.0/bandit.egg-info/requires.txt
+++ bandit-1.7.0/bandit.egg-info/requires.txt
@@ -1,3 +1,4 @@
+pbr
 PyYAML>=5.3.1
 six>=1.10.0
 stevedore>=1.20.0
--- bandit-1.7.0/requirements.txt
+++ bandit-1.7.0/requirements.txt
@@ -1,6 +1,7 @@
 # The order of packages is significant, because pip processes them in the order
 # of appearance. Changing the order has an impact on the overall integration
 # process, which may cause wedges in the gate later.
+pbr
 PyYAML>=5.3.1 # MIT
 six>=1.10.0 # MIT
 stevedore>=1.20.0 # Apache-2.0
